src/app/(BottomTabNavigation)/Profile/DeleteAccount/index.tsx

```diff
@@ -11,23 +15,46 @@ import { userInstance, User } from '../../../../types/types';
 
 function DeleteAccountScreen() {
   const [currSession, setCurrSession] = useState<User>(userInstance);
+
   useEffect(() => {
     getCurrentUserInfo().then(result => {
       setCurrSession(result);
     });
   }, []);
+
+  const deleteAccount = () => {
+    deleteCurrentUser(currSession.id);
```

Comment:
I'm not sure why but my account doesn't get deleted. Weird because you didn't change this function in this PR, right?